scripts/setup.sh

```diff
@@ -54,7 +60,7 @@ case "$ID" in
         ${PIP} --version
 
         # Use virtualenv vs system pip when we're running under Jenkins
-        if ! [ -z "${VIRT_DIR}" ] && ! [ -z "${JENKINS}" ]; then
+        if ! [ -z "${VIRT_DIR}" ] && ! [ -z "${JENKINS}" ] ; then
```

Comment:
Extra space?

Reply:
I like it like that ;) but I can remove it since I didn't make the rest of the file consistent.